pkgs/by-name/sf/sfcgal/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  cmake,
+  cgal,
+  boost,
+  mpfr,
+  gmp,
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Thanks! And also, do I directly commit the suggestions you provided, or do I make the changes on my own into one commit, or do I also add you as co-author to the commit. I'm not sure what the best practices are, or the guidelines for this repo, so it would be great to know! Thanks again for your help with this PR.